build.sbt

```diff
@@ -25,20 +31,54 @@ def jsonDependencies(scalaVersion: String) = Seq(
   logback % Test
 ) ++ jacksons ++ specsBuild.map(_ % Test)
 
+// Common settings 
+import com.typesafe.sbt.SbtScalariform._
+import scalariform.formatter.preferences._
+
+val previousVersion = Some("2.6.0-M1") // first from this separate repo
```

Comment:
@cchantep I do see your point; there is some value to knowing which APIs changed during a release. Why don't we try it, but change to `2.5.0`? The artifact name is the same so it should be easy to do that. That way we'll actually know *all* the things that broke binary compatibility in the 2.6.0 release.